echopype/convert/set_groups_ek60.py

```diff
@@ -405,46 +405,34 @@ def set_beam(self) -> xr.Dataset:
                     beam_params["beam_type"],
                     {"long_name": "type of transducer (0-single, 1-split)"},
                 ),
-                # TODO: check EK60 data spec:
-                #  the beamwidths provided are most likely 2-way beamwidth so below needs to change
-                "beamwidth_receive_alongship": (
+                "beamwidth_twoway_alongship": (
                     ["channel"],
                     beam_params["beamwidth_alongship"],
                     {
                         "long_name": "Half power one-way receive beam width along "
```

Comment:
Given the change to the variable name, shouldn't the `long_name` be changed as well?

Reply:
changed.